github/libp2p.yml

```diff
@@ -3277,6 +3277,8 @@ repositories:
     collaborators:
       admin:
         - MarcoPolo
```

Comment:
Does it need this too?

```suggestion
      admin:
        - Admin
        - w3dt-stewards
```

Reply:
We don't need it for Unified CI but I think it does make sense to add it. Let me commit the change.

Reply:
This should have gone under `teams:` not `collaborators:` 🤦  Sorry for not catching it earlier - I have fixed the config now.